src/cm_common/cm_mem_cpu_stat.c

```diff
@@ -1271,9 +1271,9 @@ extract_db_exec_stat (FILE * fp, const char *dbname, T_CM_ERROR * err_buf)
   while (fgets (linebuf, sizeof (linebuf), fp))
     {
       unsigned int *member_ptr;
-      unsigned int prop_val;
+      uint64_t prop_val;
       memset (prop_name, 0, sizeof (prop_name));
-      sscanf (linebuf, "%99s%*s%u", prop_name, &prop_val);
+      sscanf (linebuf, "%99s%*s%lu", prop_name, &prop_val);
```

Comment:
%lu is 4 bytes on windows and 8 bytes on centos.
If you want to satisfy both windows and centos, %llu seems to be good.

Reply:
Thank you for your comment!
By the way, changes to my repo are not reflected in this PR. I don't know why. So I want to close this PR and open it again.